src/main/java/Duke.java

```diff
@@ -1,4 +1,5 @@
-import duke.helper.*;
+import duke.task.*;
+import duke.exception.DukeException;
 import java.util.ArrayList;
 import java.util.Scanner;
 
```

Comment:
Can consider throwing exceptions within the operateOnTasks itself so it becomes more modular.